pkg/dom0-ztools/rootfs/etc/init.d/000-kexec

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+#
+# Kexec setup before everything else.
+#
+if cat /proc/cmdline | grep -q crashkernel; then
+    #
+    # Don't load kernel if no `crashkernel=Y@X` cmdline.
+    # This means we are on arch which does not support kdump
+    # (e.g. riscv64) or xen (see pkg/grub/rootfs.cfg)
```

Comment:
Please document the lack of support on xen etc in the markdown file.

Reply:
Will do.